### PR TITLE
fix(render): Windows compatibility for Auto rendering

### PR DESCRIPTION
**Pull request by daHaimi**

### CODE REVIEW

File: render/auto.go [hunk 1]
```diff
@@ -113,34 +114,38 @@ func (ir htmlAutoRenderer) Render(w io.Writer, data Data) error {
...
-			return ir.HTML(fmt.Sprintf("%s/new.html", templatePrefix)).Render(w, data)
+			return ir.osIndependentHTML(fmt.Sprintf("%s/new.html", templatePrefix)).Render(w, data)
 		}
 		return nil
 	}
 	cp, ok := data["current_path"].(string)
 
 	defCase := func() error {
-		return ir.HTML(fmt.Sprintf("%s/%s.html", templatePrefix, "index")).Render(w, data)
+		return ir.osIndependentHTML(fmt.Sprintf("%s/%s.html", templatePrefix, "index")).Render(w, data)
```
Comment (paganotoni) [inline render/auto.go:126]: @daHaimi could we change this to be 
```suggestion
		return ir.HTML(filepath.Join(tremplatePrefix, "index.html").Render(w, data)
```

And the same across this file?

File: render/auto.go [hunk 2]
```diff
@@ -113,34 +114,38 @@ func (ir htmlAutoRenderer) Render(w io.Writer, data Data) error {
...
 	}
 
 	if !isPlural {
-		return ir.HTML(fmt.Sprintf("%s/show.html", templatePrefix)).Render(w, data)
+		return ir.osIndependentHTML(fmt.Sprintf("%s/show.html", templatePrefix)).Render(w, data)
 	}
 	return defCase()
 }
 
+func (ir htmlAutoRenderer) osIndependentHTML(path string) Renderer {
```
Comment (paganotoni) [inline render/auto.go:145]: If we use `filepath.Join` we will not need this method.

Comment (daHaimi) [reply render/auto.go:145]: I think it is a good idea. Just KISS. :-)

**Comment by paganotoni** [comment]
@daHaimi opened #1951 as an alternative to this one, hoping to release this fix in our next version. Thank you for reporting it!

**Pull request closed without merging**